.github/CODEOWNERS

```
@@ -200,7 +200,7 @@ yarn.lock                                                @getsentry/owners-js-de
 /src/sentry/api/endpoints/organization_events_spans_histogram.py          @getsentry/performance
 /src/sentry/api/endpoints/organization_events_trace.py                    @getsentry/performance
 /src/sentry/api/endpoints/organization_events_trends.py                   @getsentry/performance
-/src/sentry/api/endpoints/organization_events_trendsv2.py                 @getsentry/performance
+/src/sentry/api/endpoints/organization_events_trends_v2.py                 @getsentry/performance
```

```suggestion
/src/sentry/api/endpoints/organization_events_trends_v2.py                @getsentry/performance
```

the smallest nit ever

---

ty!